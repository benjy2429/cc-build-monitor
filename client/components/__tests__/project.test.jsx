import React from 'react';
import { shallow } from 'enzyme';
import Project from '../project';

const defaultProps = {
  project: {
    buildStatus: 'Success',
    activity: 'Sleeping',
    name: 'org/my-project',
  },
};

describe('Project', () => {
  it('renders correctly', () => {
    const component = shallow(
      <Project {...defaultProps} />,
    );

    expect(component).toMatchSnapshot();
  });

  it('renders with the project\'s long name', () => {
    const component = shallow(
      <Project {...defaultProps} longName />,
    );

    expect(component).toMatchSnapshot();
  });
});
