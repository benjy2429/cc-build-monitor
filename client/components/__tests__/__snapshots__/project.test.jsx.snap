// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project renders correctly 1`] = `
<div
  className="project-pill project-success project-sleeping"
>
  <div
    className="project-name"
  >
    my-project
  </div>
</div>
`;

exports[`Project renders with the project's long name 1`] = `
<div
  className="project-pill project-success project-sleeping"
>
  <div
    className="project-name"
  >
    org/my-project
  </div>
</div>
`;
